packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -71,7 +71,6 @@ export const styles = css`
         padding-left: ${mediumPadding};
         ${userSelectNone}
         overflow: hidden;
-        display: flex;
```

Comment:
I moved the display configuration to the group-row implementations. They should be responsible for their self and content styling.